backend/src/main/java/com/dmytrochernykh/trainme/user_profile/entity/UserProfile.java

```diff
@@ -0,0 +1,45 @@
+package com.dmytrochernykh.trainme.user_profile.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class UserProfile {
```

Comment:
[minor] Consider using Lombok to generate boilerplate code.
https://projectlombok.org/

In this case:
```
@Getter
@Setter
@Entity
public class UserProfile { ... }
```